lambdas/sqs-message-remover/tests/test-index.js

```diff
@@ -235,7 +235,7 @@ test('sqsMessageRemover lambda updates message visibilityTimeout when workflow f
   t.is(numberOfMessages.numberOfMessagesAvailable, 0);
   t.is(numberOfMessages.numberOfMessagesNotVisible, 1);
 
-  await delay(5 * 1000);
+  await delay(6 * 1000);
```

Comment:
I don't want to use the `sleep` defined in the API package, because that would mean that this Lambda would then depend on the API package. That being said, the `delay` that is used here is _not_ in the `package.json`, and is working by coincidence. I'll fix that.

Reply:
Fixed in 227f5e2fa

Reply:
What if we put it here? https://github.com/nasa/cumulus/blob/master/packages/common/src/util.ts

It looks like @cumulus/common is required in both integration-tests and sqs-message-remover already.

Reply:
Fixed here 7d77ae8766eb904c1ea9ad00655c35cded8b6332

Reply:
Nice. Thanks for doing this! Looks good, nice to lose a minor dependency.